### PR TITLE
Add multi-date

### PR DESCRIPTION
**Pull request by alexgleith**

### CODE REVIEW

File: services/ows_refactored/surface_reflectance/ows_gm_s2_annual_cfg.py [hunk 1]
```diff
@@ -54,4 +54,5 @@
         "default_style": "simple_rgb",
         "styles": styles_gm_list,
     },
+    "multi_date": [{"allowed_count_range": [2, 4], "animate": True}],
```
Comment (SpacemanPaul) [inline services/ows_refactored/surface_reflectance/ows_gm_s2_annual_cfg.py:57]: No, it's done at the style level, not the product level.

**Comment by alexgleith** [comment]
So like this then @SpacemanPaul ?

**Review by SpacemanPaul — APPROVED** [review]
Yes, that's better.

4 frames is probably a reasonable maximum until I can implement smarter resource limits.

**Comment by SpacemanPaul** [comment]
(Note that there is no support for this yet in Terria.  It *might* work in 2D mode.)

**Pull request merged**